app/views/general_infos/edit.html.erb

```diff
@@ -107,7 +107,7 @@
   </div>
   <div class="incident-form-row">
     <div class="form-col">
-      <%= f.label :num_involved_officers, "Number of sworn officers involved", class: 'required' %>
+      <%= f.label :num_involved_officers, "Number of officers involved from #{@current_user.department}", class: 'required' %>
```

Comment:
I think there's a pending merge conflict here -- can you add the word "sworn" back into the text, before "officers"?